backends-velox/src/main/scala/org/apache/spark/sql/execution/unsafe/UnsafeColumnarBuildSideRelation.scala

```diff
@@ -366,4 +367,12 @@ case class UnsafeColumnarBuildSideRelation(
     }
     iterator.toArray
   }
+
+  override def estimatedSize: Long = {
+    if (null != batches) {
```

Comment:
nit: Can we just use `batches != null`? It's more consistent with our current coding style.